pyxem/data/simulation_fe.py

```diff
@@ -0,0 +1,91 @@
+import numpy as np
+from orix.crystal_map import Phase
+from orix.quaternion import Rotation
+from diffpy.structure import Atom, Lattice, Structure
+from diffsims.generators.simulation_generator import SimulationGenerator
+from pyxem.signals import Diffraction2D
+import skimage
+
+
+def fe_fcc_phase():
+    """Create a fe fcc phase with space group 225. This is a fcc structure with
+    a lattice parameter of 3.571 Å.
+
+    """
+    a = 3.571
+    latt = Lattice(a, a, a, 90, 90, 90)
+    atom_list = []
+    for coords in [[0, 0, 0], [0.5, 0, 0.5], [0, 0.5, 0.5], [0.5, 0.5, 0]]:
+        x, y, z = coords[0], coords[1], coords[2]
+        atom_list.append(Atom(atype="Fe", xyz=[x, y, z], lattice=latt))  # Motif part A
+    struct = Structure(atoms=atom_list, lattice=latt)
+    p = Phase(structure=struct, space_group=225)
+    return p
+
+
+def fe_bcc_phase():
+    """Create a fe bcc phase with space group 229. This is a bcc structure with
+    a lattice parameter of 2.866 Å.
+
+    """
+    a = 2.866
+    latt = Lattice(a, a, a, 90, 90, 90)
+    atom_list = []
+    for coords in [[0, 0, 0], [0.5, 0.5, 0.5]]:
+        x, y, z = coords[0], coords[1], coords[2]
+        atom_list.append(Atom(atype="Fe", xyz=[x, y, z], lattice=latt))  # Motif part A
+    struct = Structure(atoms=atom_list, lattice=latt)
+    p = Phase(structure=struct, space_group=229)
+    return p
+
+
+def fe_multi_phase_grains(num_grains=2, seed=2, size=20, recip_pixels=128):
+    bcc = fe_bcc_phase()
+    fcc = fe_fcc_phase()
+    gen = SimulationGenerator()
+    rotations1 = Rotation.random(num_grains)
+    rotations2 = Rotation.random(num_grains)
```

Comment:
I don't know if Orix supports setting the seed. In any case, these should really also be consistent between calls with the same seed, especially as correlation score distribution from templatematching are very orientation dependent

Reply:
If the distribution of the rotations does not matter, then we could simply sample e.g. Euler angles from numpy to enforce consistent RNG

Reply:
Yea, orix doesn't support a seed.  Thats a good suggestion but also something that would be useful to add to orix in the future.

Reply:
I might just leave this as is and then add that once orix supports it.